src/mtool/ui/package.json

```diff
@@ -10,6 +10,7 @@
     "@material-ui/lab": "^4.0.0-alpha.39",
     "@material-ui/pickers": "^3.2.10",
     "@material-ui/styles": "^4.11.4",
+    "@mui/material": "^5.12.1",
```

Comment:
Do we need to import a library just for table pagination which is already present in material table used in the project?